spec/unit/outputs/elasticsearch_spec.rb

```diff
@@ -59,4 +60,41 @@
       end
     end
   end
+
+  # TODO(sissel): Improve this. I'm not a fan of using message expectations (expect().to receive...)
+  # especially with respect to logging to verify a failure/retry has occurred. For now, this 
+  # should suffice, though.
```

Comment:
Might be easier to test the `#submit` method and expect a timeout exception there?